src/test/java/GmailActionableEmailsServletTest.java

```diff
@@ -49,41 +51,79 @@
 
   private static final List<String> METADATA_HEADERS = ImmutableList.of("Subject");
 
-  private static final String MESSAGE_ID_ONE = "messageOne";
-  private static final String MESSAGE_ID_TWO = "messageTwo";
+  private static final String MESSAGE_ID_OLD = "messageOne";
+  private static final String MESSAGE_ID_NEW = "messageTwo";
+  private static final String MESSAGE_ID_LOW_PRIORITY = "messageTwo";
+  private static final String MESSAGE_ID_HIGH_PRIORITY = "messageTwo";
   private static final String SUBJECT_VALUE_ONE = "subjectValueOne";
   private static final String SUBJECT_VALUE_TWO = "subjectValueTwo";
   private static final MessagePartHeader subjectHeaderOne =
       new MessagePartHeader().setName("Subject").setValue(SUBJECT_VALUE_ONE);
   private static final MessagePartHeader subjectHeaderTwo =
       new MessagePartHeader().setName("Subject").setValue(SUBJECT_VALUE_TWO);
-
-  int DEFAULT_N_DAYS = 7;
-  int NEGATIVE_N_DAYS = -1;
-
-  List<Message> SOME_MESSAGES =
+  private static final long INTERNAL_DATE_EARLY = 1;
+  private static final long INTERNAL_DATE_LATE = 2;
+
+  private static final int DEFAULT_N_DAYS = 7;
+  private static final int NEGATIVE_N_DAYS = -1;
+  private static final ActionableMessage.MessagePriority DEFAULT_PRIORITY =
+      ActionableMessage.MessagePriority.LOW;
+
+  private static final Message messageOld =
+      new Message()
+          .setId(MESSAGE_ID_OLD)
+          .setPayload(new MessagePart().setHeaders(Collections.singletonList(subjectHeaderOne)))
+          .setInternalDate(INTERNAL_DATE_EARLY);
+  private static final Message messageNew =
+      new Message()
+          .setId(MESSAGE_ID_NEW)
+          .setPayload(new MessagePart().setHeaders(Collections.singletonList(subjectHeaderTwo)))
+          .setInternalDate(INTERNAL_DATE_LATE);
+  private static final Message messageLowPriority =
+      new Message()
+          .setId(MESSAGE_ID_LOW_PRIORITY)
+          .setPayload(new MessagePart().setHeaders(Collections.singletonList(subjectHeaderOne)))
+          .setInternalDate(INTERNAL_DATE_LATE);
+  private static final Message messageHighPriority =
+      new Message()
+          .setId(MESSAGE_ID_LOW_PRIORITY)
+          .setPayload(new MessagePart().setHeaders(Collections.singletonList(subjectHeaderTwo)))
+          .setInternalDate(INTERNAL_DATE_LATE);
+
+  private static final List<Message> messagesOldestToNewest =
+      ImmutableList.of(messageOld, messageNew);
+  private static final List<Message> messagesPriorityLowestToHighest =
+      ImmutableList.of(messageLowPriority, messageHighPriority);
```

Comment:
These variables should be in constant case `MESSAGES_OLDEST_TO_NEWEST` and `MESSAGES_PRIORITY_LOWEST_TO_HIGHEST`

Reply:
Agreed! Thanks!

Reply:
RESOLVED